docs/source/tutorials/modules/basics.rst

```diff
@@ -84,7 +96,7 @@ For our ``CoulombsLaw`` module a basic constructor looks like:
 
       .. literalinclude:: ../../../../tests/cxx/doc_snippets/coulombs_law.cpp
          :language: c++
-         :lines: 36-39
+         :lines: 21-33, 36-39
```

Comment:
You don't have to change it now, but FWIW I recently learned that the `literalinclude` directive accepts `start-after` and `end-before` options so you can put tags in the source code and not have to worry about what happens when the line numbers change.